pilz_control/include/pilz_control/pilz_joint_trajectory_controller_impl.h

```diff
@@ -27,6 +27,50 @@ static constexpr double SPEED_LIMIT_NOT_ACTIVATED{ -1.0 };
 
 namespace ph = std::placeholders;
 
+/**
+ * @brief Get the Joint Acceleration Limits for each joint from the parameter server.
+ *
+ * If has_acceleration_limits is set to false acceleration limits will be set to 0.
+ *
+ * Function assumes parameter server naming prefix '/joint_limits/' for Joint Names.
+ *
+ * @param nh NodeHandle to access parameter server.
+ * @param joint_names Vector of Strings for all joint names to get the acceleration limits for.
+ * @return std::vector<double> Requested acceleration limits as vector. Has the same length as param 'joint_names'.
+ *
+ * @throw InvalidParameterException Requested values not found on param server.
+ */
+std::vector<double> getJointAccelerationLimits(const ros::NodeHandle& nh, const std::vector<std::string> joint_names)
+{
+  const std::string joint_limits_naming_prefix{ "/joint_limits/" };
```

Comment:
If I understand this correctly, the intention is to read the parameters loaded from `prbt_moveit_config`. This adds an `exec_depend` on `prbt_moveit_config` to `pilz_control`. I think this is wrong, because it introduces a circular dependency.

Reply:
Reminder: The controller limits should be higher than the planner limits.

Reply:
As agreed, we will use the limits defined in `prbt_support/config/manipulator_controller.yaml`. But we will not need a exec_depend on it, because this controller is as it is not standalone. It is only a plugin that will be loaded from `robot.launch` in `prbt_support`, anyway.

Reply:
Done.